packages/vuetify/src/util/helpers.ts

```diff
@@ -324,17 +324,17 @@ export const camelize = (str: string): string => {
   return str.replace(camelizeRE, (_, c) => c ? c.toUpperCase() : '')
 }
 
+export type FilterTreeItemFunction = (item: object, search: string, textKey: string) => boolean
```

Comment:
This should probably be somewhere else. Trying to normalize type locations in 2.0

Reply:
Agree, but i don't have a good candidate for a new location

Reply:
maybe to be safe, `types/components/VTreeview` ?